sixtyfps_runtime/corelib/model.rs

```diff
@@ -92,15 +122,13 @@ impl ModelNotify {
 impl ModelTracker for ModelNotify {
     /// Attach one peer. The peer will be notified when the model changes
     fn attach_peer(&self, peer: ModelPeer) {
-        peer.inner.upgrade().map(|rc| self.inner.borrow_mut().insert(rc));
+        if let Some(peer) = peer.inner.upgrade() {
+            self.inner().project_ref().repeaters.append(peer.as_ref())
```

Comment:
The terminology of ModelPeer and their attachment to a Model make me think that perhaps `repeaters` should be called `peers`.

Reply:
I wonder if it's worth to debug_assert here that the peer is not in the node list already?

Reply:
it might be in the list when we change model.

true, peers might be a better name